### PR TITLE
Add UpdateConfig to NodeGroup

### PR DESCRIPTION
**Pull request by mhmtsvr**
Adding `UpdateConfig` to NodeGroup, [see](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-eks-nodegroup.html#cfn-eks-nodegroup-updateconfig)

### CODE REVIEW

File: troposphere/eks.py [hunk 1]
```diff
@@ -188,6 +193,7 @@ class Nodegroup(AWSObject):
         "ReleaseVersion": (str, False),
         "RemoteAccess": (RemoteAccess, False),
         "ScalingConfig": (ScalingConfig, False),
+        "UpdateConfig": (UpdateConfig, False),
```
Comment (markpeek) [inline troposphere/eks.py:196]: Quick change for this PR, can you alphabetize this new property? Thanks!

Comment (mhmtsvr) [reply troposphere/eks.py]: can you kindly run workflows once more? thanks! @markpeek

Comment (markpeek) [reply troposphere/eks.py]: Try this if you have the black formatter installed:
```
black troposphere/eks.py
````

Comment (mhmtsvr) [reply troposphere/eks.py]: indentation issues fixed, thanks.

**Comment by markpeek** [comment]
Thanks!

**Pull request merged**